app/assets/javascripts/channels/datatables.coffee

```diff
@@ -0,0 +1,24 @@
+$(document).on 'turbolinks:load', ->
+  tableElementIds = [
+    '#passengers.admin-table'
+    '#passengers.dispatch-table'
+  ]
+  i = 0
+  while i < tableElementIds.length
+    tableElementId = tableElementIds[i]
+    if $.isEmptyObject($.find(tableElementId))
+      i++
+      continue
+    table = undefined
+    if $.fn.DataTable.isDataTable(tableElementId)
+      table = $(tableElementId).DataTable()
+    else
+      table = $(tableElementId).DataTable() #Some options would go here...
+
+    document.addEventListener 'turbolinks:before-cache', ->
```

Comment:
I think that this is the only functionality that you need here.  Might I suggest that you move it into your initialization file?